src/include/duckdb/storage/serialization/nodes.json

```diff
@@ -537,7 +537,7 @@
     "members": [
       {"id": 100,
         "name": "ignore_errors",
-        "type": "bool"
+        "type": "CSVOption<bool>"
```

Comment:
Is the serialization of `bool` identical to `CSVOption<bool>`? If not we can't change this in this manner. Could we just serialize this as a boolean? e.g. using `"serialize_property"`